users/migrations/0001_alter_user_location_country_alter_user_media_count.py

```
@@ -0,0 +1,23 @@
+# Generated by Django 5.2.7 on 2025-11-01 02:51
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('users', '0001_initial'),
+    ]
```

Ruff (0.14.3)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)